app/src/main/java/com/yalta/viewmodel/LoginViewModel.kt

```diff
@@ -3,18 +3,16 @@ package com.yalta.viewmodel
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
-import com.yalta.repositories.LoginRepo
-import com.yalta.repositories.RealLoginRepo
 import com.yalta.services.LoginService
 import kotlinx.coroutines.CoroutineDispatcher
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
+import javax.inject.Inject
 
-class LoginViewModel(
-    repo: LoginRepo = RealLoginRepo(),
-    private val dispatcher: CoroutineDispatcher = Dispatchers.IO
+class LoginViewModel @Inject constructor(
+    private val service: LoginService,
```

Comment:
Just `service`?

Reply:
yeah, why not?

Reply:
Because it's kinda general name
In similar places you've left old names